src/lumigo_tracer/spans_container.py

```diff
@@ -209,6 +212,15 @@ def add_step_end_event(self, ret_val):
             ret_val[LUMIGO_EVENT_KEY] = {STEP_FUNCTION_UID_KEY: message_id}
             get_logger().debug(f"Added key {LUMIGO_EVENT_KEY} to the user's return value")
 
+    def add_tag(self, key: str, value: str) -> bool:
+        key = str(key)
+        value = str(value)
+        tags = self.function_span["tags"]
+        if 0 < len(key) < MAX_TAG_LEN and 0 < len(value) < MAX_TAG_LEN and len(tags) < MAX_TAGS:
```

Comment:
I disagree that the value should be longer than 0 🤔

Reply:
Why? I think it should have value:
1. It makes sense that a name won't be empty
2. It might cause difficulties when the customer will want to search for it. (Not all DBs support empty strings)

Reply:
I meant specifically `value`, but it's not critical comment...
(For example, I want to tag this invocation with "important", which is a key without value...)